doc/pyproject_toml.rst

```diff
@@ -417,4 +417,35 @@ These paths:
 
 Exclusions have priority over inclusions.
 
+External data section
+---------------------
+
+Data files which your code will use should go inside the Python package folder.
+Flit will package these with no special configuration.
+
+However, sometimes it's useful to package external files for system integration,
+such as man pages or files defining a Jupyter extension. To do this, arrange
```

Comment:
perhaps

```rst
which rely on well-known paths such as ``man`` pages or defining a Jupyter `extension <https://jupyter-server.readthedocs.io/en/latest/developers/extensions.html>`_ or `kernel <https://jupyter-client.readthedocs.io/en/stable/kernels.html#kernel-specs>`_. To do this, arrange
```

> :blush: sorry, I'm sure there's a better way to do this in `.rst`...

Reply:
I'm torn on this, but for now I think I'll leave the links out. Jupyter has quite a few different extension points, and I don't want this to grow into a list of ways to extend Jupyter (not that such a list shouldn't exist, but it should be in Jupyter's own docs ;-).

Reply:
Yep, sounds good!